src/concrete/ml/sklearn/base.py

```diff
@@ -1269,16 +1270,22 @@ def __init_subclass__(cls):
                 _TREE_MODELS.add(cls)
                 _ALL_SKLEARN_MODELS.add(cls)
 
-    def __init__(self, n_bits: int):
+    def __init__(self, n_bits: int, _msb=1):
```

Comment:
I dont' think the MSB should be a configurable parameter. We should choose a good value and use that, the user shouldn't care about it. So we can make it a constant in CML somewhere

Reply:
removed